packages/next/taskfile-swc.js

```diff
@@ -100,6 +100,15 @@ module.exports = function (task) {
             keepImportAttributes,
             emitAssertForImportAttributes: keepImportAttributes,
           },
+          baseUrl: __dirname,
+          paths: {
+            'react-dom/server.edge': [
+              './compiled/react-dom-experimental/server.edge',
+            ],
+            'react-dom/static.edge': [
+              './compiled/react-dom-experimental/static.edge',
+            ],
+          },
```

Comment:
I've not seen us do this kind of aliasing for react before. Why is this required for this PR?